src/Widgets/Common/ImageBoxWidget.php

```diff
@@ -11,13 +11,30 @@
 
 class ImageBoxWidget extends BaseWidget
 {
-    protected $template = "Ceres::Widgets.Common.ImageBoxWidget";
+    protected $template = 'Ceres::Widgets.Common.ImageBoxWidget';
+    const IMAGE_EXTENSIONS = [
+        'jpg',
+        'jpeg',
+        'png',
+        'gif',
+        'svg',
+        'apng'
+    ];
+    const MODERN_IMAGE_EXTENSIONS = [
+        'jpg',
+        'jpeg',
+        'png',
+        'gif',
+        'svg',
+        'apng',
+        'webp'
+    ];
```

Comment:
`MODERN_IMAGE_EXTENSIONS` sollte eine Erweiterung der normalen extensions sein. Diese Liste doppelt zu pflegen, macht daher wenig sinn. stattdessen könntest du in den Widget-Settings dann `->withAllowedExtensions(array_merge(self:: IMAGE_EXTENSIONS, self:: MODERN_IMAGE_EXTENSIONS))` verwenden

Reply:
done.